projects/kubernetes/kubernetes/1-30/patches/0002-EKS-PATCH-Update-log-verbosity-for-node-health-and-t.patch

```diff
@@ -5,6 +5,7 @@ Subject: [PATCH] --EKS-PATCH-- Update log verbosity for node health and taint
  checks
 
 cherrypick of https://github.com/kubernetes/kubernetes/pull/121967
+For more details https://t.corp.amazon.com/V1088905731
```

Comment:
We should remove this link

Reply:
Yeah that makes sense. I'll make that change for all of those updated patches.